### PR TITLE
Release 2.9.1-beta2

### PR DESCRIPTION
**Pull request by tidy-dev**
## Description

Looking for the PR for the upcoming second beta of the v2.9.1 series? Well you've just found it, congratulations!

## Release checklist

- [x] ~~Check to see if there are any errors in Sentry that have only occurred since the last production release~~
- [x] Verify that all feature flags are flipped appropriately
---- One new feature flag and is flipped for beta
image

- [x] If there are any new metrics, ensure that central and desktop.github.com have been updated
---- No new metrics

### CODE REVIEW

**Review by sergiou87 — DISMISSED** [review]
✨

File: changelog.json [hunk 1]
```diff
@@ -1,5 +1,11 @@
 {
   "releases": {
+    "2.9.1-beta2": [
+      "[Added] Show number of lines changed in a commit - #11656",
+      "[Fixed] Made misattributed commit warning more visible in dark mode - #12210",
```
Comment (billygriffin) [inline changelog.json]:
```suggestion
      "[Improved] Increase visibility of misattributed commit warning in dark mode - #12210",
```

**Review by billygriffin — COMMENTED** [review]
One minor suggestion and otherwise 🚀

**Review by billygriffin — APPROVED** [review]
🎸

**Pull request merged**